src/main/scala/redis/api/Clusters.scala

```diff
@@ -44,7 +44,7 @@ case class ClusterSlots() extends RedisCommand[MultiBulk,Seq[ClusterSlot]] {
           case MultiBulk(Some(responses)) =>
               responses.grouped(3).map { vect =>
                 buildClusterNode(vect)
-              }.toSeq
+              }.toList
```

Comment:
`.toSeq` here returned `Stream` prior to 2.13 so I changed it to `.toList` to be compatible with all Scala versions at the same time. See the change in `RedisClusterTest.scala`.